Dev/GitHub/github-releases.30m.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# <xbar.title>GitHub Relase buddy</bitbar.title>
```

Comment:
nice idea - please provide an example picture of the plugin running.

Reply:
Added the image to the PR and to the plugin metadata! :)